code/game/objects/effects/decals/cleanable.dm

```diff
@@ -24,6 +26,11 @@
 		if(LAZYLEN(diseases_to_add))
 			AddComponent(/datum/component/infective, diseases_to_add)
 
+/obj/effect/decal/cleanable/LateInitialize()
+	if(src.loc && isturf(src.loc))

```

Comment:
remote src, call ..()

Reply:
I'm not too sure what you mean with the first part, what is remote src?